manifests/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ./rbac/cluster-role-binding.yaml
+  - ./rbac/role.yaml
+  - ./rbac/service-account.yaml
+  - service.yaml
+  - deployment.yaml
```

Comment:
I'd suggest to move these two resources under the `manager` directory to follow the usual convention.